comms/core/src/noise/mod.rs

```diff
@@ -33,3 +33,9 @@ pub use error::NoiseError;
 
 mod socket;
 pub use socket::NoiseSocket;
+use tari_utilities::{hidden_type, Hidden};
+use zeroize::Zeroize;
+
+pub(crate) const NOISE_KEY_LEN: usize = 32;
+
+hidden_type!(CommsNoiseKey, [u8; NOISE_KEY_LEN]);
```

Comment:
Recommend redefining this as a `SafeArray`.

Reply:
@sdbondi made a comment regarding the use of `SafeArray`, in this PR. I think his point was accurate, given the use of this `CommsNoiseKey`.

Reply:
Done in [PR 4994](https://github.com/tari-project/tari/pull/4994) but certainly open to discussion.